server/homebrew.api.js

```diff
@@ -259,7 +266,16 @@ const deleteGoogleBrew = async (account, id, editId, res)=>{
 	return true;
 };
 
-const deleteBrew = async (req, res)=>{
+const deleteBrew = async (req, res, next)=>{
+	try {
+		req.brew = await getBrew('edit')(req, res, next);
+	} catch (e) {
+		const { id, googleId } = getId(req);
+		console.warn(`No google brew found for id ${googleId}, the stub will be deleted.`);
+		await HomebrewModel.deleteOne({ editId: id });
+		return next();
+	}
+
 	let brew = req.brew;
 	const { googleId, editId } = brew;
```

Comment:
@jeddai I'm getting an error that it can't destructure `brew` because it's undefined. It won't let me delete *any* brew now. Can you investigate?